src/instruments/src/SD/Pages/Wheel/Wheel.tsx

```diff
@@ -0,0 +1,309 @@
+import React, { useEffect, useState } from 'react';
+import ReactDOM from 'react-dom';
+import classNames from 'classnames';
+import { SimVarProvider, useSimVar } from '@instruments/common/simVars';
+import { getRenderTarget, setIsEcamPage } from '@instruments/common/defaults';
+import { PageTitle } from '../../Common/PageTitle';
+import './Wheel.scss';
+import { HydraulicsProvider, useHydraulics } from '../../Common/HydraulicsProvider';
+import { HydraulicIndicator } from '../../Common/HydraulicIndicator';
+import { ComponentPositionProps } from '../../Common/ComponentPositionProps';
+import { EcamPage } from '../../Common/EcamPage';
+import { SvgGroup } from '../../Common/SvgGroup';
+import { Spoilers } from '../../Common/Spoilers';
+
+setIsEcamPage('wheel_page');
+
+const maxStaleness = 300;
+
+export const WheelPage = () => {
+    const temperatures: number[] = [];
+    for (let brakeNumber = 1; brakeNumber <= 4; brakeNumber++) {
+        const [temperature] = useSimVar(`L:A32NX_REPORTED_BRAKE_TEMPERATURE_${brakeNumber}`, 'celsius', maxStaleness);
+        temperatures.push(temperature);
+    }
+
+    const maxTemperatureIndex = temperatures.reduce((maxIndex, element, index, array) => (element > array[maxIndex] ? index : maxIndex), 0);
+
+    return (
+        <EcamPage name="main-wheel">
+            <PageTitle x={6} y={115} text="WHEEL" />
+
+            <HydraulicsProvider>
+                <Spoilers x={103} y={64} />
+
+                <NoseWheelSteering x={205} y={200} />
+                <LandingGearCtl x={255} y={263} />
+                <AntiSkid x={300} y={312} />
+
+                <NormalBraking x={220} y={333} />
+                <AlternateBraking x={220} y={360} />
+            </HydraulicsProvider>
+
+            <AutoBrake x={300} y={460} />
+
+            <Gear x={18} y={210} type="left" />
+            <Wheels
+                x={12}
+                y={318}
+                left={{ number: 1, temperature: temperatures[0], hottest: maxTemperatureIndex === 0 }}
+                right={{ number: 2, temperature: temperatures[1], hottest: maxTemperatureIndex === 1 }}
+            />
+
+            <Gear x={210} y={107} type="center" />
+            <WheelArch x={298} y={370} type="bottom" />
+            <WheelArch x={406} y={370} type="bottom" />
+
+            <Gear x={401} y={210} type="right" />
+            <Wheels
+                x={436}
+                y={318}
+                left={{ number: 3, temperature: temperatures[2], hottest: maxTemperatureIndex === 2 }}
+                right={{ number: 4, temperature: temperatures[3], hottest: maxTemperatureIndex === 3 }}
+            />
+        </EcamPage>
+    );
+};
+
+const NoseWheelSteering = ({ x, y }) => {
```

Comment:
Perhaps we could extract `x, y` to something like

`type TwoDimensionalPosition = { x: number, y: number }`

Reply:
Seems we are repeating it, untyped, a lot.